packages/react/src/components/Slider/Slider.tsx

```diff
@@ -907,6 +1487,17 @@ export default class Slider extends PureComponent<SliderProps> {
                   {warnText}
                 </div>
               )}
+              {correctedValue && (
+                <div
+                  role="alert"
+                  className={classNames(
+                    `${prefix}--slider__status-msg`,
+                    `${prefix}--form-requirement`
+                  )}>
+                  The inputted value &quot;{correctedValue}&quot; was corrected
```

Comment:
@tay1orjones It just occurred to me today, what is the preferred method of translating such a string? I assume it can't be hardcoded as english?

Reply:
Yeah the pattern we have established for this `translateWithId`. Here's some docs: 

https://github.com/carbon-design-system/carbon/blob/b8701c9578019c1f4fe3f92d4ef0e1a23139e9bd/docs/style.md#L430-L450